server/gui/main.py

```diff
@@ -0,0 +1,212 @@
+# flake8: noqa F403, F405
+import sys
+from os.path import splitext, basename
+
+from cefpython3 import cefpython as cef
+from PyQt5.QtCore import *
+from PyQt5.QtWidgets import *
+
+from server.gui.browser import CefWidget, CefApplication
+from server.gui.cxg_server import cellxgeneServer, DataLoadWorker, ServerRunWorker
+from server.gui.utils import WINDOWS, LINUX, MAC
+
+# Configuration
+# TODO remember this or calculate it?
```

Comment:
Suggest: start an issue to capture these UI rqmts in an epic.

Reply:
👍